src/table/components/withSelections.tsx

```diff
@@ -0,0 +1,26 @@
+import React from 'react';
+import { useContextSelector, TableContext } from '../context';
+import { getSelectionStyle } from '../utils/styling-utils';
+import { getCellSelectionState, SelectionActions, TSelectionActions } from '../utils/selections-utils';
+import { hocProps, SelectionState } from '../../types';
```

Comment:
TSelectionActions and SelectionState can be removed

Reply:
done